index.js

```diff
@@ -33,7 +33,7 @@ async function resolveVersion(version, mirror) {
 })();
 
 function runScript(shell, script, version, mirror) {
-  const child = child_process.spawn(shell, [ script, version, mirror ], { cwd: __dirname });
+  const child = child_process.spawn(shell, [ script, version, mirror ], { ...process.env, cwd: __dirname });
```

Comment:
```suggestion
  const child = child_process.spawn(shell, [ script, version, mirror ], { env: process.env, cwd: __dirname });
```
pretty sure spreading won't work

Reply:
You're absolutely right... and now I'm not sure how this made a difference to the build I tried this on.

Reply:
Thanks for pointing that out